posix.mak

```diff
@@ -473,41 +473,41 @@ dautotest: all verbatim pdf diffable-intermediaries d-latest.tag d-prerelease.ta
 # sub-directories before their parents.
 
 $W/changelog/%.html : changelog/%_pre.dd $(CHANGELOG_PRE_DDOC) $(DDOC_BIN) | $(DMD)
-	$(DDOC_BIN_DMD) -conf= -c -o- -Df$@ $(CHANGELOG_PRE_DDOC) $<
+	$(DDOC_BIN_DMD) -conf= -c -o- -transition=markdown -Df$@ $(CHANGELOG_PRE_DDOC) $<
```

Comment:
You can add this to the flags already defined at the DDOC_BIN_DMD variable. For the rest of the changes I think a DDOC_FLAGS variable makes sense.